milli/src/update/delete_documents.rs

```diff
@@ -34,6 +34,12 @@ pub struct DocumentDeletionResult {
     pub deleted_documents: u64,
     pub remaining_documents: u64,
 }
+#[derive(Debug)]
+pub struct DetailedDocumentDeletionResult {
+    pub deleted_documents: u64,
+    pub remaining_documents: u64,
+    pub used_soft_deletion: bool,
+}
```

Comment:
Can you add a little bit more documentation on this struct and the inner fields, please?

Reply:
Also could you rename the field `soft_deletion_used`?

Reply:
yes, will do 👍

Reply:
Not too much, just one or two lines would be enough :)